src/Dfc.CourseDirectory.Database/Pttcd/Views/vwProviderSearchIndex.sql

```diff
@@ -3,11 +3,19 @@ AS
 SELECT      p.ProviderId,
             p.Ukprn,
             p.ProviderName,
+            ISNULL(pc.AddressPostTown, pc.AddressItems) Town,
+            pc.AddressPostcode Postcode,
             p.ProviderStatus,
             p.UkrlpProviderStatusDescription,
-            pc.AddressPostcode Postcode,
-            ISNULL(pc.AddressPostTown, pc.AddressItems) Town,
-            p.Version -- Timestamp (RowVersion) type, used here for cosmos HighWaterMark
+            p.Version, -- Timestamp (RowVersion) type, used here for cosmos HighWaterMark
+
```

Comment:
Why lb?

Reply:
Sorry I don't understand the question, could you elaborate please

Reply:
Line break

Reply:
Oh I see. I want to group the fields after the linebreak together as a "just to fix the rankings" section. I couldn't find a way of making that clear that I liked.
Without the line break it looks like the comment only applies to the immediately following field and not all the rest. I'll have another go at making it clear without being ugly, if you have any suggestions that'd be grand.

Reply:
Or this the option of just leaving it in the git blame and not commenting it at all, but I think I'd like to highlight that these fields are only for ranking.
Another option is to add a comment to every ranking-only field, which might actually be tidier than this.

Reply:
Okay, I've pushed something I'm happer with. Thanks for the prompt. Take a look and see what you think.